meta-balena-common/recipes-core/images/balena-image.bb

```diff
@@ -38,7 +38,7 @@ IMAGE_INSTALL = " \
     "
 
 # add packages for LUKS operations if necessary
-IMAGE_INSTALL:append = "${@oe.utils.conditional('SIGN_API','','',' cryptsetup lvm2-udevrules tpm2-tools libtss2-tcti-device',d)}"
+IMAGE_INSTALL:append = "${@oe.utils.conditional('SIGN_API','','',' cryptsetup lvm2-udevrules tpm2-tools libtss2-tcti-device os-helpers-tpm2',d)}"
```

Comment:
This conditional should be on the tpm machine feature, as `SIGN_API` will also be used for non EFI/TPM device types to add secure boot functionality.